docs/component-stability.md

```diff
@@ -73,3 +73,8 @@ The component is planned to be removed in a future version and no further suppor
 ### Unmaintained
 
 A component identified as unmaintained does not have an active code owner. Such component may have never been assigned a code owner or a previously active code owner has not responded to requests for feedback within 6 weeks of being contacted. Issues and pull requests for unmaintained components will be labelled as such. After 6 months of being unmaintained, these components will be removed from official distribution. Components that are unmaintained are actively seeking contributors to become code owners.
+
+Components that were accepted based on being vendor-specific components will be marked as unmaintained if
```

Comment:
Should we say that we will attempt to reach out to the vendor to notify them of the component being marked as unmaintained?

Reply:
Added